docs/Configuration.md

```diff
@@ -14,7 +14,6 @@ language     | en, de, es, etc...                              | en            |
 calendarType | Gregorian, Islamic, Buddhist, Japanese, Persian | Gregorian     | Default calendar type for date-related web components
 [noConflict](#noConflict)  | true, false | false                            | When set to true, all events will be fired with a "ui5-" prefix only
 [formatSettings](#formatSettings)| See the [Format settings](#formatSettings) section below		| Empty object | Allows to override locale-specific configuration
-[assetsPath](#assetsPath)| See the [Assets path](#assetsPath) section below		| Empty string | Allows to set the assets path at runtime
```

Comment:
Should't we describe that it is deprecated now and put a link to the new docs, rather than delete it(as we did with the RTL)

Reply:
We want to hide it entirely and delete it soon.